deploy/kubernetes/Jenkinsfile.yml

```
@@ -11,5 +11,7 @@ when:
     tags: true                 ## for all the tags
 platform: "linux && ubuntu-18" ## default label for all the stages
 stages:
+    lint:
+       make: "make -C deploy/kubernetes all"
```

This check should be executed when `libbeat/docs/version.asciidoc` changes.

(Long-term we should probably stop commiting these files and move them to docs build)

And I guess we need to check for changes so it detects anything.
```suggestion
       make: |
         make -C deploy/kubernetes all;
         make check-no-changes;
```

---

@kuisathaverat I see you have added the trigger on changes in the version file, thanks!

But the other point in this comment about adding a check is still pending to be addressed.

---

It is https://github.com/elastic/beats/blob/master/deploy/kubernetes/Jenkinsfile.yml#L5

---

> But the other point in this comment about adding a check is still pending to be addressed.

`make -C deploy/kubernetes all` is not checking anything, I think we also need to run `make check-no-changes`.

---

yep, we need to run`make check-no-changes` also

---

Could you please add it?